net/nginx/pkg-descr

```diff
@@ -0,0 +1,9 @@
+NGINX is a high performance edge web server with the lowest memory footprint
```

Comment:
And now all uppercase? :D

Reply:
that's from ports tree. The nginx website spells it lowercase, Wikipedia too.